python/tests/test_topology.py

```diff
@@ -2894,12 +2890,15 @@ def test_small_tree_filter_zero_mutations(self):
         ts = tsutil.insert_branch_sites(ts)
         self.assertEqual(ts.num_sites, 8)
         self.assertEqual(ts.num_mutations, 8)
-        tss, _ = self.do_simplify(ts, [4, 0, 1], filter_sites=True)
-        self.assertEqual(tss.num_sites, 5)
-        self.assertEqual(tss.num_mutations, 5)
-        tss, _ = self.do_simplify(ts, [4, 0, 1], filter_sites=False)
-        self.assertEqual(tss.num_sites, 8)
-        self.assertEqual(tss.num_mutations, 5)
+        for filt in [True, False]:
+            tss, _ = self.do_simplify(ts, [4, 0, 1], filter_sites=True,
```

Comment:
This is a matter of taste, but I prefer to use hanging indents for this sort of thing:
```python
tss, _ = self.do_simplify(
    ts, [4, 0, 1], filter_sites=True, filter_unary=filt)
```
The reason I prefer hanging indents is that it's much more robust to code changes. For example, if we changed the name of the function from ``do_simplify`` to ``run_simplify``, we'd have to fix all the visual indents as well. If it was a hanging indent then no changes would be needed than a global search and replace.

For a large code-base like this, it's not a case of "if code changes are made" but "when"...

Reply:
Cool, fixed!